files/build_templates/buffers_config.j2

```diff
@@ -9,9 +9,9 @@ def
 {# Determine device topology and filename postfix #}
 {%- if DEVICE_METADATA is defined and DEVICE_METADATA['localhost']['type'] is defined %}
 {%-     set switch_role = DEVICE_METADATA['localhost']['type'] %}
-{%-     if switch_role.lower() == 'torrouter' %}
+{%-     if 'torrouter' in switch_role.lower() %}
 {%-         set filename_postfix = 't0' %}
-{%-     elif switch_role.lower() == 'leafrouter' %}
+{%-     elif 'leafrouter' in switch_role.lower() %}
```

Comment:
can you check if this will have impact on MgmtTorRouter. it should not have any buffer configuration, I mean should not have any impact. But I am not 100% sure.